### DISCUSSION TITLE
Is this client able to sync "private groups" Documents

### DISCUSSION THREAD

**Discussion by adam2392** [abraunegg/onedrive#1338] [Q&A] [answered]
Hi 

I was just wondering, if I have data stored on OneDrive associated with an account, does it get data for: something like that `https://livejohnshopkins.sharepoint.com/sites/private_group`, where `private_group` is a Shared Library folder?

**Comment by abraunegg** [comment, marked as answer]
@adam2392 

Shared Libraries are supported as per: https://github.com/abraunegg/onedrive/blob/master/docs/SharePoint-Shared-Libraries.md

Closing issue as not a bug, next time please ask a question via [Discussions](https://github.com/abraunegg/onedrive/discussions)
